src/lib/controllers/sso_sp.php

```diff
@@ -1,56 +1,59 @@
 <?php
   // Setup Default Options Array
-  
-  if (isset($_POST['submit']) && wp_verify_nonce($_POST['_wpnonce'],'sso_sp') ) 
-  {    
+
+  if (isset($_POST['submit']) && wp_verify_nonce($_POST['_wpnonce'],'sso_sp') )
+  {
+      $upload_dir = constant('SAMLAUTH_CONF') . '/certs/' . get_current_blog_id();
```

Comment:
@ktbartholomew I left this in to retain compatibility if you were using the ini file previously, the plugin doesn't write except on config save.